grunt/helpers.js

```diff
@@ -196,11 +196,30 @@ module.exports = function(grunt) {
         var includes = grunt.config('includes');
         var excludes = grunt.config('excludes');
 
+        var isIncluded;
+        var isExcluded;
+
         // carry on as normal if no includes/excludes
         if (!includes && !excludes) return true;
 
-        var isIncluded = includes && pluginPath.search(exports.getIncludedRegExp()) !== -1;
-        var isExcluded = excludes && pluginPath.search(exports.getExcludedRegExp()) !== -1;
+        // get plugins directory within theme/less
+        var compareVar = grunt.file.doesPathContain((grunt.config('sourcedir') + 'theme' + path.sep + grunt.config('theme') + path.sep + 'less' + path.sep + 'src' + path.sep + 'plugins'), pluginPath);
+
+        // if plugin directory within theme is in the filepath
+        if (compareVar) {
+
+            // plugins dir is in the file path so check against the list of includes
+            var bits = [];
+            bits = pluginPath.split("/"); // split the file path
+            var match = bits[bits.length - 2]; // get plugin directory name
```

Comment:
Does the -2 here assume here that there will be no further directories beneath the plugins folder?

Reply:
It does. This isn't an argument for it but I am not sure there should be other directories within a plugins folder for less (unless there was to be say an index.less, that included multiple less files from within a sub-directory).

Do you have a recommended better approach I could attempt, for stripping out the plugin name from the filePath?